src/core_plugins/kibana/server/tutorials/netflow/index.js

```diff
@@ -13,7 +13,6 @@ export function netflowSpecProvider() {
       ' indexes the events into Elasticsearch, and installs a suite of Kibana dashboards.' +
       ' This module support Netflow Version 5 and 9.' +
       ' [Learn more]({config.docs.logstash}/netflow-module.html).',
-    //iconPath: '', TODO
```

Comment:
Is Netflow not going to get an icon? Ever? I noticed that it wasn't in the screenshot either. @alexfrancoeur?

Reply:
@ycombinator I'm open to any icon here. I've pinged about screenshots a few times now too, would love to see one in there. Think you could add one? We could use a Cisco icon for Netflow, I can't find any others. We have a few things like system metrics / logs that do not have an icon either.

Reply:
Yeah, IIRC I used the Cisco icon in the modules UI prototype. If we can, I think that's the closest one we can use here. But I'm not sure about the legality of using it (I'd asked about this in an earlier issue: https://github.com/elastic/dev/issues/837#issuecomment-339625666). 

So I'd say Cisco if legal is good with it or the generic/default icon if not.

Reply:
As for screenshots, I don't know how to create Netflow data w/o an actual Cisco device or something. I'm sure there is a way to simulate it thought. @jsvd any ideas?

Reply:
@ycombinator I've ran icons by legal previously, I think we're good here. Let's add the Cisco icon. I'll link to it in the issue